onnxruntime/core/providers/coreml/builders/model_builder.cc

```diff
@@ -906,6 +906,7 @@ Status ModelBuilder::SaveModel() {
 
 #if defined(COREML_ENABLE_MLPROGRAM)
   if (create_ml_program_) {
+    // we need to jump through some hurdles to get the model path the ML Program load wants.
```

Comment:
very minor nit: "jump through some hoops"

Reply:
you clearly haven't seen me hurdle... 🤣